src/utils/traverseHeadings.ts

```diff
@@ -6,6 +6,7 @@ export function traverseHeadings(tree, filterKey: string): string[] {
   for (const node of tree) {
     if (typeof node !== 'object') continue;
     if (!('props' in node)) continue;
+    if ('mdxType' in node.props && node.props.mdxType === 'BlockSwitcher')continue; // skip parsing the BlockSwitcher, we don't want BlockSwitcher headings in the TOC
```

Comment:
nit: could we add spacing between the `if` statement and the `continue`? It looks like it's right after the parenthesis